src/Test/TestCombat.java

```diff
@@ -29,10 +30,10 @@ private void setupTest()
     GameScenario scn = new GameScenario();
     testCo1 = new Strong(scn.rules);
     testCo2 = new Patch(scn.rules);
-    Commander[] cos = { testCo1, testCo2 };
+    Army[] cos = { new Army(testCo1), new Army(testCo2) };
 
     testMap = new MapMaster(cos, MapLibrary.getByName("Firing Range"));
-    testGame = new GameInstance(testMap);
+    testGame = new GameInstance(cos, testMap);
```

Comment:
Why not keep around the basic `GameInstance(MapMaster)` constructor and just auto-create the armies for simple/standard situations with no tags? Would avoid boilerplate in most places we create games.

Reply:
The `MapMaster` requires armies as input anyway.
It's certainly *possible* to do what you're suggesting, but we'd need some sort of interim structure to hold the stuff the map wants to dump into the armies... and that's probably more work.